helsinki/README.md

```diff
@@ -1,202 +1,211 @@
 ## Meetups organized in Helsinki
 
-#### The leadership team
+#### Organizers
 
- - Lucas Käldström ([@luxas](https://github.com/luxas)), CNCF Ambassador & Kubernetes maintainer, lucas@luxaslabs.com
+ - Lucas Käldström ([@luxas])(https://github.com/luxas), CNCF Ambassador, luxas labs, lucas@luxaslabs.com
```

Comment:
This seems to be a bug in the template. It should render as a link, I don't think it does now